playbooks/openstack/openshift-cluster/launch.yml

```diff
@@ -55,7 +55,7 @@
     register: stack_show_status_result
```

Comment:
Is it possible to have the heat command block on creating the stack? If so, then it might be better to run that as an async task and poll for the completion/result instead of performing the retries here...